specs/linux_file_system_hierarchy.md

```diff
@@ -231,16 +231,19 @@ use `$libdir` (see below), instead.
 
 ### `/usr/libexec/`
```

Comment:
hmm, what about adding some qualifier here to the section title like `(facultative)` or `(optional)` or `(dispensable)` or so? That would mean it would already in the TOC be visible as optional

Reply:
See the third commit.